includes/polling/os/fortiwlc.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use LibreNMS\RRD\RrdDefinition;
+
+$version = trim(snmp_get($device, "MERU-GLOBAL-STATISTICS-MIB::mwSystemGeneralVersion.0", "-OQv"), '"');
```

Comment:
You don't need to trim() these others have done that as a legacy thing which hasn't been cleaned up.

Also, it would be better if you got all the data in one operation with snmp_get_multi(), there should be some other os polling files that use that you can look at for an example.